community/kernel/src/main/java/org/neo4j/kernel/impl/locking/LockClientAlreadyClosed.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.locking;
+
+/**
+ * Exception that will be thrown in case when closed {@link org.neo4j.kernel.impl.locking.Locks.Client}
+ * will be used to acquire shared/exclusive lock
+ */
+public class LockClientAlreadyClosed extends RuntimeException
```

Comment:
Perhaps it's silly, but for consistency sake I like exceptions to have a name that ends with "Exception".